nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -1,20 +1,20 @@
-# Test Procedures
+# Scenarios
```

Comment:
Could you rename this file from TEST_PROCEDURES.md to SCENARIOS.md?

Reply:
All done